Pod/Tests/TestStash.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+import XCTest
```

Comment:
I think it's easier to describe intent and test case with `Quick` but like I did in this case, its faster to setup without it seeing as it is already included with the core frameworks.